erc1155/src/base.rs

```diff
@@ -1,6 +1,7 @@
 use gstd::{prelude::*, ActorId};
 
 use crate::common::BalanceOfBatchReply;
+use crate::common::TokenMetadata;
 
 pub trait ERC1155TokenBase {
     fn balance_of(&self, account: &ActorId, id: &u128) -> u128;
```

Comment:
I'd add some type alias for `id` for distinguishing it from the `amount`/`value`.

```rust
type SomeId = u128;
```

```suggestion
    fn balance_of(&self, account: &ActorId, id: &SomeId) -> u128;
```

Reply:
Should it also be done for the lib.rs file?

Reply:
Yes, I'd introduce a type alias at every place we use this id.